i18n/utils.py

```diff
@@ -94,5 +94,5 @@ def storage(cls):
         if cls._storage is None:
             storage = getattr(settings, 'I18N_STORAGE', 'django.core.files.storage.FileSystemStorage')
             storage_cls = import_string(storage)
-            cls._storage = storage_cls(location=getattr(settings, 'I18N_STORAGE_LOCATION', 'i18n'))
+            cls._storage = storage_cls(location=getattr(settings, 'I18N_STORAGE_LOCATION', 'i18n/static'))
```

Comment:
I'd suggest we have a goal of entirely getting rid of either `I18N_STATIC_DIR` or of `I18N_STORAGE_LOCATION`, rather than just updating them both to incidentally have the same values